.github/workflows/check-code.yml

```diff
@@ -13,192 +13,288 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  get-commit-head:
-    name: Get HEAD commit message
+  # get-commit-head:
```

Comment:
is it intended, to comment them all out?

Reply:
Nope. It was for testing. I reverted that.